package-lock.json

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
what's the purpose of having both package and package-lock?
why are there more entries in the package-lock than in package?

Reply:
package-lock contains exact versions of all dependencies and sub(.*)-dependencies, so you can have reproducible builds. package.json only contains the top-level ones, the ones you need.